### PR TITLE
Rename instances of master with main

### PR DESCRIPTION
**Pull request by robertwu1**
Now that we have officially removed the master branch, we should change instances of master to main.

Without this change, there will be an extra automatic link redirection from master to main by Github.

### CODE REVIEW

**Comment by philburk** [comment]
What are "instances of master"?
Are these links in the markdown docs?

File: apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 // Based on the WaveFileWriter in Java from the open source JSyn library by Phil Burk
-// https://github.com/philburk/jsyn/blob/master/src/com/jsyn/util/WaveFileWriter.java
+// https://github.com/philburk/jsyn/blob/master/src/main/java/com/jsyn/util/WaveFileWriter.java
```
Comment (philburk) [inline apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h:18]: JSyn is still using a "master" branch.

**Review by philburk — CHANGES_REQUESTED** [review]

File: apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h (hunk 1 above)
Comment (robertwu1) [reply apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h:18]: Yup, hence this is https://github.com/philburk/jsyn/blob/master/src/main/java/com/jsyn/util/WaveFileWriter.java and not https://github.com/philburk/jsyn/blob/master/src/master/java/com/jsyn/util/WaveFileWriter.java

Comment (philburk) [reply apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h:18]: Ah! So there is a "main" folder in the "master" branch in JSyn. Got it.

**Review by philburk — APPROVED** [review]

**Pull request merged**